src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.cpp

```diff
@@ -141,7 +141,7 @@ std::unique_ptr<WriteBufferFromFileBase> AzureObjectStorage::writeObject( /// NO
     return std::make_unique<WriteIndirectBufferFromRemoteFS>(std::move(buffer), std::move(finalize_callback), object.absolute_path);
 }
 
-void AzureObjectStorage::listPrefix(const std::string & path, RelativePathsWithSize & children) const
+void AzureObjectStorage::findAllFiles(const std::string & path, RelativePathsWithSize & children) const
```

Comment:
Is this implementation correct?

Reply:
I think so, but I have not checked.

Reply:
I leaved it since it can be used for `send_metadata`